src/desktop/components/inquiry_questionnaire/views/account.coffee

```diff
@@ -17,11 +23,17 @@ module.exports = class Account extends StepView
     'click button': 'submit'
     'click .js-mode': 'change'
     'click .js-iq-save-skip': 'next'
+    'change #accepted_terms_of_service': 'checkAcceptedTerms'
+    'click #signup-fb': 'fbSignup'
 
   initialize: ({ @user, @inquiry, @artwork, @state, @modal }) ->
     @modal?.dialog 'bounce-in'
     @active = new Backbone.Model mode: 'auth'
 
+    # remove after gdpr compliance a/b test closes
+    splitTest('gdpr_compliance_test').view()
```

Comment:
I'm not sure if this is a good place to initiate an A/B test. Why not doing this on server-side?

Reply:
From my understanding this doesn't initialize the test rather [tracks the experiment](https://github.com/artsy/force/tree/master/src/desktop/components/split_test#tracking-experiments). The test gets initialized on the [server by default](https://github.com/artsy/force/tree/master/src/desktop/components/split_test#getting-the-outcome-of-a-split-test-on-the-server) I believe. I could be wrong, maybe @mzikherman or @kanaabe could clarify.

Reply:
Yep!

We generally call the `.view()` client-side, since that sends the actual 'Experiment Viewed' event to Segment (along with other enriched client-side data).